examples/config.nsq.minimal.hocon

```diff
@@ -0,0 +1,32 @@
+# The minimum required config options for loading from nsq
+"input": {
+  "type": "nsq"
+  "streamName": "test-nsq-stream"
+  "channelName": "test-nsq-channel-name"
+  "nsqlookupdHost": "127.0.0.1"
+  "nsqlookupdPort": 34189
+}
+"output": {
+  "good": {
+    "type": "elasticsearch"
+    "client": {
+      "endpoint": "localhost"
+      "port": 9200
```

Comment:
I'm not strongly opinionated on this, but just suggestions.  The following all seem like reasonable defaults which could get moved to the application.conf file instead of the minimal hocon:

- port 9200 for elasticsearch.  This is a very standard port.
- `ssl: false`
- `signing: false`
- All of the `type` fields, e.g. `input.type`.

The reason I suggest this... imagine if you want to run in an environment where you cannot easily mount a directory with a hocon file.  The command line will look like:

```bash
docker run snowplow/elasticsearch-loader:2.0.0 \
  -Dsnowplow.input.type=nsq \
  -Dsnowplow.input.streamName=test-nsq-stream \
  -Dsnowplow.input.channelName=test-nsq-channel-name \
  # and so on and so on
```

So more convenient if we can reduce the number of command line parameters.

Reply:
We can do that but I am not sure whether that would be too far. They are not like buffer size or Kinesis maximum records. They can easily know and set port number or whether ssl is true or not. I am not strongly opinionated neither, I can make necessary changes if you want.

However, I am not sure how we will set `input.type`s in the `aplication.conf`. We can do that for `output.good.type`  since it will be most probably `elasticsearch` but I am not sure what to set `input.type` and `output.bad.type` as default.

Reply:
You are correct about `input.type` -- it does not make sense to set this (I was wrong).  And I agree with your suggestion to set `output.good.type` to `elasticsearch` because afterall this is the elasticsearch loader!

Let's seek out other people's opinions about `ssl` and `signing` and `port` - hopefully we can find someone with a stronger opinion than either of us!!

Reply:
If we set defaults, then anyone who wants to change them has to provide their own values. If we don't set defaults, then everyone has to provide their own values.

It seems to me that adding defaults can only help and there is no downside to it.